lib/utils/checkAgainstRule.js

```diff
@@ -22,7 +23,7 @@ module.exports = function(
   if (!callback) throw new Error("checkAgainstRule requires a callback");
   if (!options.ruleName)
     throw new Error("checkAgainstRule requires a 'ruleName' option");
-  if (!rules[options.ruleName])
+  if (rules.indexOf(options.ruleName) === -1)
```

Comment:
I'd suggest `(!rules.includes(options.ruleName))` as a clearer alternative.

Reply:
For something reason, I'd gotten it into my head that `.includes` was introduced in node 8... [that's not the case](https://node.green/#ES2016-features-Array-prototype-includes).

I'll update now.